### ISSUE TITLE
fork / new version

### ISSUE THREAD

**Issue by lukeapage** [bfricka/less-preview#32]
Hi @bfricka,

I'm having a go at writing a new version - https://github.com/less/less-preview - using Aurelia. Its partly because I want to investigate writing an app in Aurelia. I've started from scratch, but stolen your design (so most of the css and html). I've also gone with having a gulp task that generates a importable array of less versions, then loading the files from the CDN.. I think this should make updating easier and means we can host on github pages.

Hope all of that is okay! At the moment you cannot switch versions or change options and I haven't deployed to gh-pages yet (might need to wait a week for Aurelia to release bundling cli). I can give you write access if you'd like, but I guess you are too busy to contribute.

Once I'm on a feature-par maybe we can discuss moving the domain the point at the new version?

Thanks!

**Comment by lukeapage** [comment]
I put it on gh-pages as-is (lots of issues but anyway) : http://lesscss.org/less-preview/
until bundling comes along its going to take an age to load.

**Comment by lukeapage** [comment]
now much improved.
http://lesscss.org/less-preview/

**Comment by bfricka** [comment]
Awesome dude. Yeah we can totally look at transferring the domain target. I took a bit of a break from all this, sorry. Aurelia looks amazing! Before I went on hiatus, I was looking deeply at jspm and ES6, and it was kind of a pain to get everything quickly bootstrapped. I'm super stoked to start playing with this.

Let me know when you feel like you're ready and I'll point the domain (although I don't know exactly how).

**Comment by lukeapage** [comment]
@bfricka I made you an admin.

We made lesscss.org a published apex domain but I hit problems trying to get lesscss.org to resolve to less.github.io/less-docs.. so in the end I configured less.github.io so that the domains could be straight forward aliased.

At the moment this creates a strange effect where by http://lesscss.org/less-preview/ works.

What I'd try is following https://help.github.com/articles/tips-for-configuring-an-a-record-with-your-dns-provider/ and https://help.github.com/articles/my-custom-domain-isn-t-working/
So.. creating a CNAME file in the root of the gh-pages branch, then pointing the DNS record over to github pages.

We could do that sometime next week or weekend? That gives me this weekend to polish off a few things on the site. It would be good to be coordinated as it will probably go wrong at least for a bit, taking down the existing site.
